include/klee/klee.h

```diff
@@ -260,6 +260,8 @@ void klee_trace_extra_ptr(void *ptr, int width, char *name, char *type,
                           char *prefix, TracingDirection td);
 void klee_trace_extra_fptr(void *ptr, int width, char *name, char *type,
                            char *prefix, TracingDirection td);
+void klee_begin_instruction_tracing();
```

Comment:
Do we still need this?
I thought we had just introduced it as a hacky attempt, and removed it later

Reply:
Resolved in a commit that I just added - Should I remove this hack code from klee completely or let it be in case it ever needs to be used again or referred to?

Reply:
Remove completely